README.md

```diff
@@ -143,26 +152,21 @@ Deploy
 This section outlines the most common options for deployment.
 Use ``invoke -h <command>`` to learn more about any invoke command.
 
-Once your tests pass you can make a new build for the project you want to deploy:
-
-```bash
-invoke aws.build <target-project-name>
-```
-
-After you have created the image you can push it to AWS.
-This command will push to a registry that's available to all environments on AWS:
-
-```bash
-invoke aws.push <target-project-name>
-```
-
-To see a list of all currently available images for a project you can run:
+Before deploying you'll want to decide on a version number.
+To see a list of all currently available images for a project and the versions they are tagged with you can run:
 
 ```bash
 invoke aws.print-available-images <target-project-name>
```

Comment:
maybe clarify a bit with an example, not sure what to use here <target-project-name> ( I think service and harvester)